Week5/Solution_week-5/WEEK_5_ASSIGNMENT-checkpoint.ipynb

```diff
@@ -0,0 +1,184 @@
+   "metadata": {},
+   "source": [
+    "### For inputting 5 records from user"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "fc996bc0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter Student ID : 106\n",
+      "Enter Student Name : Shiv\n",
+      "Enter Student City : Rajkot\n",
+      "Enter Student Contact : 8899776655\n",
+      "Enter Student ID : 107\n",
+      "Enter Student Name : Gopi\n",
+      "Enter Student City : Vadodara\n",
+      "Enter Student Contact : 8866554433\n",
+      "Enter Student ID : 108\n",
+      "Enter Student Name : Laxmi\n",
+      "Enter Student City : Surat\n",
+      "Enter Student Contact : 9911223344\n",
+      "Enter Student ID : 109\n",
+      "Enter Student Name : Prathana\n",
+      "Enter Student City : Valsad\n",
+      "Enter Student Contact : 778899000123\n",
+      "Enter Student ID : 110\n",
+      "Enter Student Name : Saraswati\n",
+      "Enter Student City : Bardoli\n",
+      "Enter Student Contact : 9012678545\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in range(5):\n",
+    "    sid=int(input('Enter Student ID : '))\n",
+    "    sname=input('Enter Student Name : ')\n",
+    "    city=input('Enter Student City : ')\n",
+    "    contact=int(input('Enter Student Contact : '))\n",
+    "    input_list.append((sid,sname,city,contact))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4267180",
+   "metadata": {},
+   "source": [
+    "### Writing the CSV file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "460d55ac",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('student.csv','w',newline='') as csvfile:\n",
+    "    write_obj=csv.writer(csvfile)\n",
+    "    header=['S_ID','S_NAME','S_CITY','S_CONTACT_NO']\n",
+    "    write_obj.writerow(header)\n",
+    "    write_obj.writerows(input_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "158d917e",
+   "metadata": {},
+   "source": [
+    "### Reading the CSV file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "1643927a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['S_ID', 'S_NAME', 'S_CITY', 'S_CONTACT_NO']\n",
+      "['101', 'OM', 'Bardoli', '9012365432']\n",
+      "['102', 'Sai', 'Surat', '9012345678']\n",
+      "['103', 'Ram', 'Vadodara', '8200123455']\n",
+      "['104', 'Radha', 'Navsari', '9925778989']\n",
+      "['105', 'Krishna', 'Jamnagar', '9099226754']\n",
+      "['106', 'Shiv', 'Rajkot', '8899776655']\n",
+      "['107', 'Gopi', 'Vadodara', '8866554433']\n",
+      "['108', 'Laxmi', 'Surat', '9911223344']\n",
+      "['109', 'Prathana', 'Valsad', '778899000123']\n",
+      "['110', 'Saraswati', 'Bardoli', '9012678545']\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open('student.csv','r',) as csvfile:\n",
```

Comment:
Without giving path, where is this file locate in pc?!

Reply:
The default path for any file when we save it without giving specific location will be :

C:\users\user_name\filename.extension_of_file

Here, 
         user_name depends on that in which user account you have logged in... 
And then it follows the filename with extension. Here which is student.csv.. 


Thank you..